presto-parser/src/main/java/io/prestosql/sql/tree/CreateSchema.java

```diff
@@ -28,23 +28,25 @@
     private final QualifiedName schemaName;
     private final boolean notExists;
     private final List<Property> properties;
+    private final Optional<PrincipalSpecification> principal;
```

Comment:
s/principal/authorization?

Reply:
I was internally debating this. What we *are* referring to is the principal, so I thought this is better,

Reply:
Let's leave it this way, it's also aligned with SetSchemaAuthorization.